app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -136,6 +136,10 @@ description: |
   elsif nic_delay
     options.push("nicdelay=#{nic_delay} linksleep=#{nic_delay}")
   end
+
+  if host_param('append_kernel_options')
```

Comment:
@lzap does the name of the param make sense?
Do we already document the params somewhere, where we should add it?

Reply:
This snippet is included in PXE templates:

```
app/views/unattended/provisioning_templates/PXELinux/kickstart_default_pxelinux.erb
38:APPEND initrd=<%= @initrd %> <%= pxe_kernel_options %> <%= snippet("kickstart_kernel_options").strip %>
```

As you can see, there is a helper called `pxe_kernel_options` defined in `app/models/operatingsystem.rb` which then takes kernelcmd host parameter. It is a bug tricky to see, but it is documented in our macros.